environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
I just want to make sure that anything passing in `node7` that's not explicitly failing in `node75` is implicitly passing in `node75`.